sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +210,38 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private static ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+            var connectionName = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
+
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = connectionName;
```

Comment:
Using line 216 here directly -

```suggestion
            blobDetails.Connection = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;;
```